lib/DBIx/TempDB.pm

```diff
@@ -287,7 +295,7 @@ and L<Mojo::mysql>.
 
 =cut
 
-sub url { shift->{url} }
+sub url { shift->{url}->uri }
```

Comment:
A URI::db created URL is a nested url starting the db: scheme, which will not be usable by Mojo::Pg or Mojo::mysql. The ->uri accessor makes sure that only the inner URL is returned (the one starting with mysql: or pg: etc).